dynamic_instrumentation/src/instrument.rs

```diff
@@ -7,23 +7,23 @@ use log::debug;
 use rustc_index::vec::Idx;
 use rustc_middle::mir::visit::{MutatingUseContext, PlaceContext, Visitor};
 use rustc_middle::mir::{
-    BasicBlock, BasicBlockData, Body, BorrowKind, Local, LocalDecl, Location, Operand, Place,
-    PlaceElem, Rvalue, SourceInfo, Terminator, TerminatorKind, START_BLOCK,
+    BasicBlock, BasicBlockData, Body, BorrowKind, HasLocalDecls, Local, LocalDecl, Location,
+    Operand, Place, PlaceElem, Rvalue, SourceInfo, Terminator, TerminatorKind, START_BLOCK,
 };
 use rustc_middle::ty::{self, TyCtxt, TyS};
-use rustc_span::def_id::{DefId, DefPathHash, CRATE_DEF_INDEX};
-use rustc_span::{Symbol, DUMMY_SP};
+use rustc_span::def_id::{DefId, DefPathHash};
+use rustc_span::DUMMY_SP;
 use std::collections::HashMap;
 use std::fs::File;
-use std::mem;
 use std::path::Path;
 use std::sync::Mutex;
 
 use crate::arg::{ArgKind, InstrumentationArg};
 use crate::cast::cast_ptr_to_usize;
 use crate::deref::{has_outer_deref, remove_outer_deref, strip_all_deref};
-use crate::into_operand::IntoOperand;
-use crate::point::{InstrumentationAdder, InstrumentationPoint};
+use crate::hooks::Hooks;
+use crate::point::apply::InstrumentationApplier;
```

Comment:
can `point` re-export `apply` to have `crate::point::InstrumentApplier`, which matches `crate::point::InstrumentationAdder`?

Reply:
Sure, but I don't think it's too important.  They're usually just auto-imported.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/560/commits/1e7f2a09b4a18319d50e668a97d1d46f612f0099.